store/proxy/proxy_store.go

```diff
@@ -568,3 +578,40 @@ func (s *Store) fromInternal(apiContext *types.APIContext, schema *types.Schema,
 
 	return data
 }
+
+// getListStruct returns a runtime object for storing results from list requests.  If the Store's scheme does not return
+// a type for the resource associated with the store, a generic type will be used.
+func (s *Store) getListStruct() runtime.Object {
+	// try to find the list type for this store
+	obj, err := s.typer.New(schema.GroupVersionKind{
+		Group:   s.group,
+		Version: s.version,
+		Kind:    s.kind + "List",
+	})
+
```

Comment:
nit: remove line. Even with comments, I think the error check should be next to the call that may produce it.